### PR TITLE
fix: change filecoin storage to web3 storage in client

### PR DESCRIPTION
**Pull request by vasco-santos**

### CODE REVIEW

**Comment by vasco-santos** [comment]
The API test is failling everywhere, but not locally for me. We need to figure this out tomorrow

**Comment by olizilla** [comment]
Agree, issue is here for CI fails https://github.com/web3-storage/web3.storage/issues/11

File: packages/client/README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ _This module is under developement and is not published yet._
 ## Usage
 
 ```js
-import FilecoinStorage from 'web3.storage'
+import { WebStorage } from 'web3.storage'
```
Comment (olizilla) [inline packages/client/README.md:12]:
```suggestion
import { Web3Storage } from 'web3.storage'
```

File: packages/client/README.md [hunk 2]
```diff
@@ -9,10 +9,10 @@ _This module is under developement and is not published yet._
...
 
 // Construct with token and endpoint
-const client = new FilecoinStorage({ token: apiKey })
+const client = new WebStorage({ token: apiKey })
```
Comment (olizilla) [inline packages/client/README.md:15]:
```suggestion
const client = new Web3Storage({ token: apiKey })
```

**Review by olizilla — CHANGES_REQUESTED** [review]
renaming stuff is harder than it sounds

**Review by olizilla — APPROVED** [review]

**Pull request merged**